File: questao_01/questao_01.java
class Carro {
    private String marca;
    private String modelo;
    private int ano;

    
    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    
    public void exibirDetalhes() {
        System.out.printf("Carro: %s %s, Ano: %d%n", marca, modelo, ano);
    }

    public static void main(String[] args) {
        Carro carro1 = new Carro("Peugeot", "208", 2021);
        Carro carro2 = new Carro("Caoa Chery", "Tiggo 7 Pro", 2020);
        Carro carro3 = new Carro("Jeep", "Renegade", 2019);

        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}

File: questao_01/questao_01.go
package main

import (
	"fmt"
)

type Carro struct {
	marca  string
	modelo string
	ano    int
}

func (c Carro) ExibirDetalhes() {
	fmt.Printf("Carro: %s %s, Ano: %d\n", c.marca, c.modelo, c.ano)
}

func main() {
	carro1 := Carro{"Peugeot", "208", 2021}
	carro2 := Carro{"Caoa Chery", "Tiggo 7 Pro", 2020}
	carro3 := Carro{"Jeep", "Renegade", 2019}

	carro1.ExibirDetalhes()
	carro2.ExibirDetalhes()
	carro3.ExibirDetalhes()
}
